docs/Plugins/refdiff.md

```diff
@@ -4,10 +4,9 @@ description: >
   RefDiff Plugin
 ---
 
-
 ## Summary
 
-For development workload analysis, we often need to know how many commits have been created between 2 releases. This plugin calculates which commits differ between 2 Ref (branch/tag), and the result will be stored back into the database for further analysis.
+For development workload analysis, we often need to know how many commits have been created between 2 releases. This plugin calculates which commits differ between 2 commits, and the commits are produced by Ref (branch/tag) or by deployments, and the result will be stored back into the database for further analysis.
```

Comment:
Text change:
RefDiff is a plugin that performs calculation tasks and has 2 main purposes.
- Calculate the difference in commits between `releases/tags` to [analyze the amount of code in each release](add an anchor link)
- Calculate the difference in commits between `deployments` to [calculate DORA metrics](add an anchor link)

The output of RefDiff is stored in the table [refs_commits_diff](https://devlake.apache.org/docs/DataModels/DevLakeDomainLayerSchema#refs_commits_diffs).